doc/source/tutorial/stats/sampling.rst

```diff
@@ -0,0 +1,180 @@
+.. _non-uniform-random-number-sampling:
+
+=====================================================
+Universal Non-Uniform Random Number Sampling in SciPy
```

Comment:
we should also mention the relationship to the distributions in `scipy.stats`: these already have the rvs method. The general aim is to provide a fast generation method in rv_continuous. The tools presented here allow the user to experiment with different methods that can lead to performance improvements in certain situations, e.g., if a large number of samples is needed or if particular shape parameters are used.

+ that while the same uniform rng is used, the resulting rvs will be different in general

Reply:
> we should also mention the relationship to the distributions in `scipy.stats`: these already have the rvs method. The general aim is to provide a fast generation method in rv_continuous. The tools presented here allow the user to experiment with different methods that can lead to performance improvements in certain situations, e.g., if a large number of samples is needed or if particular shape parameters are used.
> 
> and mention that while the same uniform rng is used, the resulting rvs will be different in general

Ah, yes! I forgot to include this. This will be definitely useful. I will update the tutorial to include this.

Reply:
Done.

Reply:
Thanks for adding a clarification:

> Please note the difference between the ``rvs`` method of the distributions present in :mod:`scipy.stats` and the one provided by these generators: The general aim of these generators is to provide a fast generation method in :class:`~rv_continuous`.

I propose to change it slighly since at the moment, the new UNU.RAN generators are not used by `rv_continuous` at all. Of course, this might change once we find a good way to use them in that class. My comment above was misleading.

Please note the difference between the ``rvs`` method of the distributions present in :mod:`scipy.stats` and the one provided by these generators. In general, the aim of the generation methods implemented in :class:`~rv_continuous` is to provide generators that work reasonably well for common use cases, e.g., to generate small samples using different shape parameters or to generate large samples for a fixed shape parameter. Depending on the goal, specialized methods can lead to a substantial speedup. For example, at the cost of a rather expensive setup step, very fast generation can be achieved for large samples and a fixed shape parameter.

Reply:
This sounds great! I will change it this way.

Reply:
I have changed this according to your comment.